com.hazelcast.jet.server.JetCommandLineTest missing jar for submit failures

_master_ (commit ffa4e319c96fcc2a0c3950a21cdafb1f8dea9246)

These tests failed on custom build but this branch had no difference between current master (see aforementioned commit ID to see current master):
* `com.hazelcast.jet.server.JetCommandLineTest.test_submit_withWrongClassname`
* `com.hazelcast.jet.server.JetCommandLineTest.test_submit_clientShutdownWhenDone`
* `com.hazelcast.jet.server.JetCommandLineTest.test_submit`
* `com.hazelcast.jet.server.JetCommandLineTest.test_submit_argsPassing`
* `com.hazelcast.jet.server.JetCommandLineTest.test_submit_nameUsed`
* `com.hazelcast.jet.server.JetCommandLineTest.test_submit_withClassname`

Failed on OracleJDK8: http://jenkins.hazelcast.com/view/Jet/job/jet-custom-oss-oracle-jdk8/1/testReport/

Stacktrace is the similar for all test, this one is for `test_submit`:
```
picocli.CommandLine$ExecutionException: Error while calling command (public void com.hazelcast.jet.server.JetCommandLine.submit(com.hazelcast.jet.server.JetCommandLine$Verbosity,java.lang.String,java.lang.String,java.lang.String,java.io.File,java.util.List) throws java.lang.Exception): java.lang.Exception: File /tmp/testjob-7967980873267045967.jar could not be found.
	at picocli.CommandLine.execute(CommandLine.java:1207)
	at picocli.CommandLine.access$800(CommandLine.java:141)
	at picocli.CommandLine$RunAll.handle(CommandLine.java:1416)
	at picocli.CommandLine$RunAll.handle(CommandLine.java:1376)
	at picocli.CommandLine$AbstractParseResultHandler.handleParseResult(CommandLine.java:1243)
	at picocli.CommandLine.parseWithHandlers(CommandLine.java:1526)
	at com.hazelcast.jet.server.JetCommandLine.runCommandLine(JetCommandLine.java:157)
	at com.hazelcast.jet.server.JetCommandLineTest.run(JetCommandLineTest.java:515)
	at com.hazelcast.jet.server.JetCommandLineTest.test_submit(JetCommandLineTest.java:425)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:114)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.Exception: File /tmp/testjob-7967980873267045967.jar could not be found.
	at com.hazelcast.jet.server.JetCommandLine.submit(JetCommandLine.java:200)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at picocli.CommandLine.execute(CommandLine.java:1189)
	... 20 more
```

Standard output:
```
Hiccups measured while running test 'test_submit(com.hazelcast.jet.server.JetCommandLineTest):'
07:53:05, accumulated pauses: 562 ms, max pause: 179 ms, pauses over 1000 ms: 0
07:53:10, accumulated pauses: 4217 ms, max pause: 1720 ms, pauses over 1000 ms: 1
07:53:15, accumulated pauses: 2015 ms, max pause: 548 ms, pauses over 1000 ms: 0
07:53:20, accumulated pauses: 1638 ms, max pause: 174 ms, pauses over 1000 ms: 0
07:53:25, accumulated pauses: 38 ms, max pause: 14 ms, pauses over 1000 ms: 0
```

--- a/hazelcast-jet-all/src/test/java/com/hazelcast/jet/server/JetCommandLineTest.java
+++ b/hazelcast-jet-all/src/test/java/com/hazelcast/jet/server/JetCommandLineTest.java
@@ -125,7 +125,7 @@ public class JetCommandLineTest extends JetTestSupport {
     }
 
     public boolean isJarFileExists() {
-        return testJobJarFile.toFile().exists();
+        return Files.exists(testJobJarFile);
     }
 
     @After

--- a/hazelcast-jet-all/src/test/java/com/hazelcast/jet/server/JetCommandLineTest.java
+++ b/hazelcast-jet-all/src/test/java/com/hazelcast/jet/server/JetCommandLineTest.java
@@ -83,10 +83,14 @@ public class JetCommandLineTest extends JetTestSupport {
 
     @BeforeClass
     public static void beforeClass() throws IOException {
-        testJobJarFile = Files.createTempFile("testjob-", ".jar");
-        IOUtil.copy(JetCommandLineTest.class.getResourceAsStream("testjob.jar"), testJobJarFile.toFile());
+        createJarFile();
         xmlConfiguration = new File(JetCommandLineTest.class.getResource("hazelcast-client-test.xml").getPath());
         yamlConfiguration = new File(JetCommandLineTest.class.getResource("hazelcast-client-test.yaml").getPath());
+    }
+
+    public static void createJarFile() throws IOException {
+        testJobJarFile = Files.createTempFile("testjob-", ".jar");
+        IOUtil.copy(JetCommandLineTest.class.getResourceAsStream("testjob.jar"), testJobJarFile.toFile());
     }
 
     @AfterClass
@@ -112,6 +116,16 @@ public class JetCommandLineTest extends JetTestSupport {
         sourceMap = jet.getMap(SOURCE_NAME);
         IntStream.range(0, ITEM_COUNT).forEach(i -> sourceMap.put(i, i));
         sinkList = jet.getList(SINK_NAME);
+        assertTrueEventually(() -> {
+            if (!isJarFileExists()) {
+                createJarFile();
+            }
+            assertTrue(isJarFileExists());
+        });
+    }
+
+    public boolean isJarFileExists() {
+        return testJobJarFile.toFile().exists();
     }
 
     @After

